scrapy/settings/default_settings.py

```diff
@@ -230,9 +230,12 @@
 STATS_DUMP = True
 
 STATSMAILER_RCPTS = []
-
-TEMPLATES_DIR = abspath(join(dirname(__file__), '..', 'templates'))
-
+TEMPLATES_DIR_BASE = [
```

Comment:
@nramirezuy : What if we keep only the first line in TEMPLATE_DIR_BASE and we modify the `get_template_dir` function, to look through three stages:
1) `TEMPLATES_PROJECT` -- template folder input given by user
2) For `startproject`, we look at `abspath(join(dirname(__file__), '..', 'templates'))` [folder where we run the command]. For `genspider`, we find closest `scrapy.cfg` and use its directory.
3) `TEMPLATES_PROJECT_BASE`  -> the original scrapy template folder.

Please let me know what do you think.